spec/requests/api/v1/items_request_spec.rb

```diff
@@ -36,4 +36,89 @@
 
     expect(item[:data][:attributes]).to have_key(:name)
   end
+
+  it "can get one item by id" do
+    id =  create(:item).id
+
+    get "/api/v1/items/#{id}"
+
+    expect(response).to be_successful
+
+    item = JSON.parse(response.body, symbolize_names: true)
+
+    expect(item[:data][:id]).to eq("#{id}")
+
+    expect(item[:data]).to have_key(:id)
+    expect(item[:data]).to have_key(:type)
+    expect(item[:data]).to have_key(:attributes)
+
+    expect(item[:data][:attributes]).to have_key(:name)
+  end
+
+  it "can create a new item" do
+    merchant_id = create(:merchant).id
+
+    headers = { "CONTENT_TYPE" => "application/json"}
+    post "/api/v1/items", :params => {
+      item: {
+        name: "Toy Plane",
+        description: "Fly your own airplane",
+        unit_price: 22.35,
+        merchant_id: merchant_id
+        }
+      }
+
+    expect(response).to be_successful
+
+    new_item = JSON.parse(response.body, symbolize_names: true)
+    expect(new_item[:data]).to have_key(:id)
+    expect(new_item[:data]).to have_key(:type)
+    expect(new_item[:data]).to have_key(:attributes)
+
+    expect(new_item[:data][:attributes]).to have_key(:name)
+  end
+
+  it "can update an existing item" do
+    id = create(:item).id
+
+    headers = { "CONTENT_TYPE" => "application/json"}
+    patch "/api/v1/items/#{id}", :params => {
+      item: {
+        name: "Tennis Ball"
+        }
+      }
+
+    expect(response).to be_successful
+    item = Item.find_by(id: id)
+    expect(item.name).to_not eq("Stuffed Giraffe")
+    expect(item.name).to eq("Tennis Ball")
+
+    new_item = JSON.parse(response.body, symbolize_names: true)
+    expect(new_item[:data]).to have_key(:id)
+    expect(new_item[:data]).to have_key(:type)
+    expect(new_item[:data]).to have_key(:attributes)
+
+    expect(new_item[:data][:attributes]).to have_key(:name)
+  end
+
+  it "can destroy an item" do
+    item = create(:item)
+
+    expect(Item.count).to eq(1)
+
+    delete "/api/v1/items/#{item.id}"
+
+    expect(response).to be_successful
+
+    expect(Item.count).to eq(0)
```

Comment:
Testing both here that the item was deleted and that the appropriate JSON response is being returned.